dash/dg2-column-sorting-api (#21637)
highcharts/highcharts: docs/datagrid/columns.md

## Sorting

> The DataGrid allows you to set the querying options. The basic one is sorting.

Reviewer: I believe this line can be deleted

> Sorting object consists of two main configuration options:

Reviewer: ```suggestion
The optional `sorting` object consists of two configuration options:
```

> - **`sortable`**: A boolean that determines whether a column can be sorted by the user clicking on the column header.

Reviewer: ```suggestion
- **`sortable`**: A boolean that determines whether a column can be sorted by the end user clicking on the column header.
```

> - **`order`**: Specifies the initial sorting order for a column. It can be set to `'asc'` (ascending), `'desc'` (descending), or `null`, which indicates that initial sorting is disabled for that column. Note that only one column can have an `order` set at a time; if multiple columns are specified, only the last one will be considered.

Reviewer: ```suggestion
- **`order`**: Specifies the initial sorting order for a column. It can be set to `'asc'` (ascending) or `'desc'` (descending). If `order` is defined in multiple columns only the last one will be considered.
```
